sdks/java/extensions/sorter/pom.xml

```diff
@@ -28,33 +28,11 @@
 
   <artifactId>beam-sdks-java-extensions-sorter</artifactId>
   <name>Apache Beam :: SDKs :: Java :: Extensions :: Sorter</name>
-  
+
   <properties>
-    <hadoop.version>2.7.1</hadoop.version>
+    <hadoop.version>2.7.3</hadoop.version>
```

Comment:
I wanted to align the version with the one that we are using in the hadoop components and hdfs, but I will remove this since it is unrelated to this PR.